### PR TITLE
[OpenLoop-2.1.0] patch up missing lib for 10_2_X

### PR DESCRIPTION
**Pull request by SiewYan**
continue the Openloop lib sage part1

### CODE REVIEW

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
@SiewYan , looks like you are minssing
```
Source1: openloops-user.coll
```

**Comment by SiewYan** [comment]
Thanks @smuzaffar , its fixed now for both PR.

**Comment by smuzaffar** [comment]
please test

**Comment by smuzaffar** [comment]
+externals

File: openloops.spec [hunk 1]
```diff
@@ -24,6 +24,7 @@ EOF
 export SCONSFLAGS="-j %{compiling_processes}"
 ./openloops update --processes generator=0
 ./openloops libinstall lhc.coll
+./openloops libinstall %{_sourcedir}/openloops-user.coll
```
Comment (fabiocos) [inline openloops.spec:27]: @SiewYan is the line above still needed?

**Comment by fabiocos** [comment]
+1

File: openloops.spec (hunk 1 above)
Comment (SiewYan) [reply openloops.spec:27]: No, you may remove 

./openloops libinstall lhc.coll

**Pull request merged**